03-11-2018/HybernateExample/src/main/resources/hibernate.config.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!DOCTYPE hibernate-configuration PUBLIC
+"-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+"http://hibernate.sourceforge.net/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+
+	<session-factory>
+
+		<property name="show_sql">true</property>
+		<property name="connection.driver_class">com.mysql.jdbc.Driver</property>
```

Comment:
write a ech property description in comment why it is using like "<property name="connection.driver_class">com.mysql.jdbc.Driver</property>" initialise database driver class